bundles/AdminBundle/Resources/public/js/pimcore/document/hardlink.js

```diff
@@ -96,6 +96,14 @@ pimcore.document.hardlink = Class.create(pimcore.document.document, {
         var tabTitle = this.data.key;
         this.tabPanel = Ext.getCmp("pimcore_panel_tabs");
         var tabId = "document_" + this.id;
+
+        var iconClass;
```

Comment:
Maybe we should move this code snippet to the abstract, as it is the same for every type.